src/App.tsx

```diff
@@ -1,16 +1,26 @@
-import './App.scss';
+import { useState } from 'react';
+
+import { Movie } from './types/Movie';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
+import './App.scss';
+
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const handleMovieAddition = (movie: Movie) => {
+    setMovies([...movies, movie]);
```

Comment:
```suggestion
    setMovies(prevMovies => [...prevMovies, movie]);
```
To be sure that we are using the most recent version of `movies`